Client/OnboardingNotificationCardHelper.swift

```diff
@@ -0,0 +1,20 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class OnboardingNotificationCardHelper: FeatureFlaggable {
+    var cardPosition: OnboardingNotificationCardPosition {
+        return featureFlags.getCustomState(for: .onboardingNotificationCard) ?? .noCard
```

Comment:
How come this has a default, given that nimbus has a default? Theoretically, we shouldn't.

Reply:
Yes, it does, doesn't it. Hmm. Good point. I totally forgot. :)